src/app/pages/credentials/certificates-dash/certificate-edit/certificate-edit.component.html

```diff
@@ -15,6 +15,14 @@
             [required]="true"
             [tooltip]="helptext.edit.name.tooltip | translate"
           ></ix-input>
+          <ix-input
+            *ngIf="form.controls?.renew_days"
```

Comment:
I feel like the `controls` array would always exist. Maybe this check should be on the form or not at all there.

Reply:
I do not check for array, I check for **renew_days** control presence. It is working, please recheck.

Reply:
The question mark `?` is checking whether the controls array exists. That's what I was talking about. Not the ngIf as a whole.